### DISCUSSION TITLE
What is your favorite noodle??

### DISCUSSION THREAD

**Discussion by daisuke02090** [NAIST-SE/AttractingContributors#1052] [General]
I really want to know what your favorite noodles are.
Please help me.
https://github.com/daisuke02090/AttractingContributors-noodles

**Comment by kozmr** [comment]
Hi, I contributed to your project.
Could you contribute to my project?

https://github.com/kozmr/AttractingContributors-YouTube
Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1113

**Reply by daisuke02090** [reply]
Thank you!!
I made a PR on your project!

**Comment by Ryota-keda** [comment]
I just made a PR on your repo!
Could you contribute to my repo?
https://github.com/Ryota-keda/Baseball

**Reply by daisuke02090** [reply]
Thank you!!
I added Michihiro Ogasawara

**Comment by anamhamza786** [comment]
Hey,
I sent you a pull request. can you please send me a pull request.

Please Add your Favourite Players name here.
Here is the link to my forked repository: https://github.com/anamhamza786/AttractingContributors-cricketplayers.git
